### ISSUE TITLE
lr_scheduler()

### ISSUE THREAD

**Issue by kyk37** [pytorch/pytorch#127884]
### 📚 The doc issue

https://pytorch.org/docs/stable/optim.html#module-torch.optim.lr_scheduler
When clicking on this link it takes you to a blank, bottom of the page section of torch.optim. The link for this function inside torch.optim just under the section "How to adjust learning rate", also links to this empty section...

Here is the plain link to the page w/o the stuff after .html
https://pytorch.org/docs/stable/optim.html

### Suggest a potential alternative/fix

Please make a section regarding how this function works on this page, and what parameters can go into the function optim.lr_scheduler(). 

cc @svekars @brycebortree @vincentqb @jbschlosser @albanD @janeyx99 @crcrpar

**Comment by janeyx99** [comment]
May be addressed by #128248 @spzala could you confirm?

**Comment by spzala** [comment]
> May be addressed by #128248 @spzala could you confirm?

@janeyx99 sure, will do, thanks!

**Comment by spzala** [comment]
@janeyx99 hi, my PR related to LR Scheduler changes has been merged, https://github.com/pytorch/pytorch/pull/128679 but I don't see those changes are fixing this particular issue. I could reproduce this issue, and though I don't have fix in mind right now, I can work on it if no one else's working. Thanks!

**Comment by janeyx99** [comment]
@spzala yes, we'd accept a fix for this!
